tests/copy-paste.js

```diff
@@ -685,13 +685,13 @@ define([
             paste([
                 ["id", "type", "labelItext:en-default", "itemsetData"],
                 ["/select", "SelectDynamic", "select",
-                 '[{"instance":null,"nodeset":"/items","labelRef":"@name","valueRef":"@id"}]'],
+                 '[{"instance":null,"nodeset":"/items","labelRef":"name","valueRef":"@id"}]'],
```

Comment:
>  I'm not sure which part of the code should be taking them into account.

I think `afterDynamicSelectInsert` is what should be taking them into account. https://github.com/dimagi/Vellum/pull/636/files/9a7b64b1e16d100b18c094fcc944471a13cfe7bb#diff-f15dee15f58a804357f7fca0280de97dR374 does take it into account, but only if there are no data sources.

> copying a DynamicSelect without children

This is copying it with a child. The "child" in the form builder isn't actually a new question/mug, it just displays what's in `mug.p.itemsetData`